pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentSubscription.java

```diff
@@ -1336,6 +1336,7 @@ public CompletableFuture<SubscriptionStatsImpl> getStatsAsync(GetStatsOptions ge
         }
         subStats.msgBacklogNoDelayed = subStats.msgBacklog - subStats.msgDelayed;
         subStats.msgRateExpired = expiryMonitor.getMessageExpiryRate();
+        subStats.msgExpiredCounter = expiryMonitor.getMessageExpiryCount();
```

Comment:
could this be named `msgExpired` so that the name is consistent with the existing other field names?